napari/layers/points/points.py

```diff
@@ -368,7 +368,7 @@ def __init__(
         data, ndim = fix_data_points(data, ndim)
 
         # Indices of selected points
-        self._selected_data = set()
+        self._selected_data = Selection()
```

Comment:
Did removing it affect tests? Sometimes I've seen stuff like this was needed because the `__init__` triggered some layertype-specific methods that assumed the existence of the attribute, but then we still needed to initialize it properly afterwards?

Reply:
Removing it here will still pass all tests.

Reply:
I went ahead and removed it.